### ISSUE TITLE
List supported properties/functionality in README file

### ISSUE THREAD

**Issue by treyhunner** [RReverser/github-editorconfig#8]
Does this plugin only support indentation (in both view and edit mode)?

If so indent_style/indent_size (and possibly tab_width) should be listed as supported in the README file (similar to the way the other plugins list support).

**Comment by RReverser** [comment]
Nope, it supports all the properties except as for `charset` and `end_of_line` which are controlled by Github anyway.
